### PR TITLE
Wfp 568

### PR DESCRIPTION
**Pull request by carlov20**

### CODE REVIEW

File: app/services/workers/process-import.js [hunk 1]
```diff
@@ -22,11 +23,12 @@ module.exports.execute = function (task) {
...
   const tasks = []
   let workloadReportId
-
   return insertWorkloadReport()
-    .then(function (insertedWorkloadReportId) {
-      workloadReportId = insertedWorkloadReportId
-      return populateStagingCourtReporters()
+    .then(function (insertedWorkloadReport) {
+      workloadReportId = insertedWorkloadReport.id
+      return updateCloseOpenWorkloadReports(insertedWorkloadReport.effective_from).then(function () {
```
Comment (markreesmoj) [inline app/services/workers/process-import.js:29]: I think this will close the workload report we have just inserted as well as the previous one

File: app/services/data/check-for-duplicate-court-reports.js [hunk 2]
```diff
@@ -1,9 +1,14 @@
 const knex = require('../../../knex').appSchema
+const workloadReportStatus = require('../../constants/workload-report-status')
```
Comment (markreesmoj) [inline app/services/data/check-for-duplicate-court-reports.js:2]: Is this change still necessary?

File: app/services/data/check-for-duplicate-workloads.js [hunk 3]
```diff
@@ -1,9 +1,14 @@
 const knex = require('../../../knex').appSchema
+const workloadReportStatus = require('../../constants/workload-report-status')
```
Comment (markreesmoj) [inline app/services/data/check-for-duplicate-workloads.js:2]: Is this change still necessary?

File: app/services/data/check-for-duplicate-court-reports.js (hunk 2 above)
Comment (carlov20) [reply app/services/data/check-for-duplicate-court-reports.js]: reverted the change

File: app/services/data/check-for-duplicate-workloads.js (hunk 3 above)
Comment (carlov20) [reply app/services/data/check-for-duplicate-workloads.js]: reverted the change

**Review by markreesmoj — APPROVED** [review]
